### ISSUE TITLE
Accessibility: Reset to Default

### ISSUE THREAD

**Issue by EvanSchallerer** [osu-cass/SampleItemsWebsite#112]
"Reset to Default" does not reset dropdown to default.
repro:

- Open accessibility modal
- Set "color choices" to "Yellow on Blue"
- Click "Update"
- Open accessibility modal
- Click "Reset to Default"
- Open accessibility modal
    - "Color Choices" still says "Yellow on Blue"

**Comment by RikkiGibson** [comment]
We're failing to clear component state on the accessibility modal when "reset to default" is clicked.

**Comment by RikkiGibson** [comment]
https://github.com/osu-cass/SampleItemsWebsite/pull/113
